packages/debugger/test/data/returnvalue.js

```diff
@@ -279,8 +287,9 @@ describe("Return value decoding", function () {
     try {
       await instance.fail(); //web3 throws on failure
     } catch (error) {
-      txHash = error.hashes[0]; //it's the only hash involved
+      txHash = error.receipt.transactionHash;
     }
+    if (!txHash) assert.fail("should have errored and set txHash");
```

Comment:
Given that we're using Chai, I think it would make more sense to just write this line as
```assert.isDefined(txHash, "should have errored and set txHash")```
(That also applies throughout this file, I won't tag each individual one.)

Reply:
Sure!